ideas/322-dao.md

```diff
@@ -0,0 +1,76 @@
+---
+id: #322-Dao
+title: DAO
+status: research
+lead-contributor: Barry @bgits
+contributors:
+    - Barry @bgits
+    - Iuri @iurimatias
+    - Jarrad @jarradhope
+    - Ricardo @3esmit
+    - Richard @richard-ramos
+budget:
+- actual: xxx
+- estimate: yyy
+- currency: ETH/USD/SNT
+---
+
+
+DAO
+=
+
+## Summary and Goals
+
+One major drawback in legacy social networks is the lack of influence their users possess over the networks themselves. They are often powerless in having a say on how the platform evolves. We aim to democratise this power, giving stakeholders a direct influence over all decisions within the network, including how the software is developed with voting and allocation of funds via liquid pledging
+
+
+## Contributors
+
+- Barry @bgits
+- Iuri @iurimatias
+- Jarrad @jarradhope
+- Ricardo @3esmit
+- Richard @richard-ramos
+
+(+ Janitor swarm)
+
+## Communication
+
+- `status channel`: #status-dao
+- `sync schedule`: brainstorming on Mondays / stand ups on Wednesday
+- `pivotal link`: https://www.pivotaltracker.com/n/projects/2231548
+- `research notes`: #status-dao
+- `github repo`: 
+- - https://github.com/status-im/liquidpledging/tree/embark-dapp
+- - https://github.com/status-im/snt-voting
+
+## Research
+
+- Consider using Aragon for DAO related functionality. Currently Aragon does not support the delegation chain used by Liquid Pledge, however this is something that could be built by us. This also promotes collaboration between organizations, reuse of expertise, and avoids reinventing the wheel.
+
+- Liquid Pledge uses Solidity < `0.5.0`. Migrating to 0.5.0 would make debugging easier thanks to revert exceptions including an error message. However, the project uses OpenZeppelin and Aragon's contracts which would make the migration to `0.5.0` more complicated. 
+
+#### Key assumption to test:
+- Funding done on the mainnet. A goal is to have funds currently in the multisig of the GMBH to start moving to DAO’s for allocation to projects.
+- Voting is described as part of governance in the Status whitepaper. The voting dapp is currently used as a signaling tool, and it does not have in-chain consequences. Will voting be used also for project funding? (via topic democracy or other mechanism)
+- Principles signing should be required in order to access the funds. A plugin could be made for this, or in case identities are implemented, maybe a Claim could be created for an identity once the principles are signed.
+
```

Comment:
I would add to this section: 
-Are delegates needed? 
- Is directly funding a specific purpose Aragon DAOs from the primary multisig easier to manage?